src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int base;
+    private int heigh;
+
+    public RightTriangle(int base, int heigh, Color color) {
+        super(color);
+        this.base = base;
+        this.heigh = heigh;
+    }
+
+    @Override
+    public double getArea() {
+        return (base * heigh) / 2;

```

Comment:
The calculation for the area should be cast to double to ensure precision, as integer division may lead to loss of precision. Consider using '(double) base * heigh / 2' to ensure the result is a double.